doc/README-ZTEX

```diff
@@ -0,0 +1,80 @@
+---------
+Overview
+---------
+If you have ZTEX boards USB-FPGA 1.15y, you can use them with JtR.
+Available "formats" are descrypt-ztex and bcrypt-ztex, more are planned.
+
+-------------
+How to build
+-------------
+To build JtR bleeding-jumbo with ZTEX 1.15y board support, install
+libusb (e.g., the libusb-devel package on Fedora) in addition to jumbo's
+usual dependencies.  Then use "./configure --enable-ztex".  The rest of
+the build is as usual for jumbo.
+
+---------------
+Usage on Linux
+---------------
+To access a ZTEX board as non-root (and you shouldn't build nor run JtR
+as root) on a Linux system with udev, add this:
+
+ATTRS{idVendor}=="221a", ATTRS{idProduct}=="0100", SUBSYSTEMS=="usb", ACTION=="add", MODE="0660", GROUP="ztex"
+
+e.g. to /etc/udev/rules.d/99-local.rules (create this file).  Then issue
+these commands as root:
+
+groupadd ztex
+usermod -a -G ztex user # where "user" is your non-root username
+systemctl restart systemd-udevd # or "service udev restart" if without systemd
+
+In order to trigger udev to set the new permissions, (re)connect the
+device after this point.
+
+-----------------
+Usage on Windows
+-----------------
+It requires WinUSB driver to access the board.
+You can install the driver using Zadig 2.2 software.
+
+----------------------
+Format Specific Notes
+----------------------
+descrypt-ztex. That's a fast "format", USB 2.0 is unable to transfer
+password candidates from the host at the rate they are computed.
+Use mask mode to allow on-board candidate generation.
+Other feature is the limit of no more than 2,047 hashes per salt.
```

Comment:
Should add a note that mask mode may also be used along with another mode, with `?w`.

The limit isn't really a "feature". Better wording would be "Also note that there's a limit ..."